include/zephyr/net/net_stats.h

```diff
@@ -623,6 +623,12 @@ struct net_stats_wifi {
 
 	/** Total number of unicast packets received and sent */
 	struct net_stats_pkts unicast;
+
+	/** The current maximum PHY rate of transfer of data*/
+	net_stats_t current_max_rate;
```

Comment:
This does not feel very statistics like variable. Would using #76062 give better real time information for the rate?

Reply:
`wifi_iface_status` is a better place for this, it's mostly depends on `enum wifi_link_mode link_mode;` and capabilities of the peer.